mmstudio/src/main/java/org/micromanager/internal/dialogs/ConfigDialog.java

```diff
@@ -321,7 +298,8 @@ public void initializePropertyTable() {
    public void dispose() {
       studio_.events().unregisterForEvents(this);
       super.dispose();
-      savePosition();
+      WindowPositioning.setUpLocationMemory(this, this.getClass(), null);
```

Comment:
`setUpLocationMemory` (or should it be `setUpBoundsMemory`?) needs to be called before showing the window (often in the constructor when using a `JDialog` subclass) and never in methods that get called when the window closes.

In this case, that appears to be done in the subclasses of `ConfigDialog`, so it should not be needed here.